docs/Versions.md

```diff
@@ -92,9 +92,30 @@ MsQuic->SetParam(
     &Settings);
 ```
 
+### Configuring Versions via Windows Registry
+MsQuic supports setting the Acceptable Versions, Offered Versions, and Fully-Deployed Versions lists via the Windows registry. These settings are global for all servers and clients on the machine.
+The registry settings are overridden by settings specified in the code.
+The registry values must be created under the HKLM\System\CurrentControlSet\Services\MsQuic\Parameters key.
+Each list is stored in the registry as a `REG_BINARY` type, with the version numbers in little-endian (host) order.
+The registry value for Acceptable Versions must be named "AcceptableVersions".
+The registry value for Offered Versions must be named "OfferedVersions".
+The registry value for Fully-Deployed Versions must be named "FullyDeployedVersions".
+
+Here's a sample .reg file that creates all three lists with QUIC version 2 first and QUIC version 1 after, in little endian order, and enables version negotiation:
+```reg
+Windows Registry Editor Version 5.00
+
+[HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\MsQuic\Parameters]
+"AcceptableVersions"=hex:cf,43,33,6b,01,00,00,00
```

Comment:
I know it doesn't work this way today, but I think it's worth updating the settings code (in a future PR) to handle a more simpler configuration model, where only one of these _has_ to be implemented; including the `Enabled` one. Thoughts?

Reply:
I think the risk of a single-list is that a deployment is made using the single-list model, and then we add QUIC 3 support, and the deployment either does the version rollout wrong, or the infrastructure isn't built to do the rollout, so they have to spend more time updating infrastructure and testing it to even do a rollout.

Reply:
I do think that setting any version list via registry should enable the version negotiation extension without the extra registry value.